sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -101,6 +101,18 @@ def test_feature_importances():
     mask = np.abs(transformer.estimator_.coef_) > 1e-5
     assert_array_equal(X_new, X[:, mask])
 
+    # Linear model with CSR yields same result as with ndarray
+    lest = LinearSVC()
```

Comment:
I wasn't sure what convention to follow, since `est` was previously instantiated to a RandomForestClassifier, so I made a new variable to go with the new classifier I was introducing. I can standardize them if you think it's more clear.

Also, should I change all of the classifiers in `test_feature_importances` to `clf`?